_dashboards/dashboard/index.md

```diff
@@ -0,0 +1,163 @@
+- The **panel** (E) allows you to add existing visualizations to the dashboard or create new ones for the dashboard.
+
+
+_Panel_ is a term used to refer to a visualization displayed on a dashboard. The terms _panel_ and _visualization_ may be used interchangeably throughout this and other Dashboard-related documentation.
+{: .note}
+
+## Adding sample data
+
+If you haven't already added the OpenSearch Dashboards sample datasets as described in [Quickstart guide for OpenSearch Dashboards](https://opensearch.org/docs/latest/dashboards/quickstart-dashboards/), add them before proceeding to the next steps. **Sample eCommerce orders** is used for this tutorial. 
+
+To add the sample dataset:
+
+1. On the OpenSearch Dashboards **Home** page, choose **Add sample data**.
+2. Choose **Add data** and choose **Sample eCommerce orders**.
+
+## Creating dashboards
+
+Now that you've added the sample data, you can create a dashboard using that data. The sample dataset has existing sample visualizations, and you can use those visualizations or create new visualizations for the dashboard. For this tutorial, you'll do both.
+
+1. From the navigation panel, choose **Dashboard**.
+2. From the **Dashboards** window, choose **Create Dashboard**.
+3. Set the time filter to **Last 30 days**.
+4. Choose **Add an existing**.
+5. From the **Add panels** window, choose **Create new** and **Visualization**. This is a shortcut for creating a dashboard and a visualization in one flow. Alternatively, you can use the **Visualize** application to create visualizations and then add them to the dashboard.
+6. From the **New Visualization** window, choose **[eCommerce] Average Sold Quantity**. The donut chart is added to the dashboard.
+
+You've created a basic dashboard like the following, which you'll continue using throughout this tutorial.
+
+![Creating a basic dashboard]({{site.url}}{{site.baseurl}}/images/dashboards/dashboard-basic.png)
+
+## Creating and saving visualizations
+
+Continuing with the dashboard you created in the preceding steps, you'll create a new visualization and save it to the dashboard.
+
+1. From the dashboard toolbar, choose **Create new**.
+1. From the **New Visualization** window, choose **Vertical Bar** and then select the index pattern **opensearch_dashboards_sample_data_ecommerce**. You'll see an image like the following.
+
+    ![New visualization vertical bar chart]({{site.url}}{{site.baseurl}}/images/dashboards/new-bar-viz.png)
+
+1. Configure the chart's **Data** settings by verifying **Metrics** is **Y-axis Count**.
+1. From the **Buckets** panel, choose **Add** and select **X-axis** from the **Add Bucket** dropdown list.
+1. From the **Aggregation** dropdown list, select **Date histogram**.
+1. Choose **Update**. You'll see a bar chart visualization like the following:
+
+    ![Configuring a visualization]({{site.url}}{{site.baseurl}}/images/dashboards/configure-bar-viz.png)
+
+1. From the toolbar, select **Save**
```

Comment:
These steps are a bit of a tangent from actually learning about dashboards. For the purpose of the tutorial, it might be better to just save the default vis, without any configurations, so they see the flow of dashboard->vis->dashboard with out extraneous steps.

Reply:
Revised.

Reply:
Revised.